cdncheck.go

```diff
@@ -53,3 +53,28 @@ func (c *Client) Check(ip net.IP) (matched bool, value string, itemType string,
 	}
 	return false, "", "", err
 }
+
+// GetDefaultProviders exports default providers
+func GetDefaultCdnProviders() []string {
+	var provider []string
+	for k := range generatedData.CDN {
```

Comment:
We can package the map keys extraction into a helper function (e.g. `mapKeys`).
Another alternative can be to bump the required go version to 1.18 and use the new function `maps.Keys` from the standard library (ref https://pkg.go.dev/golang.org/x/exp/maps#Keys)

Reply:
Used helper function